docs/logs/sending-docker-logs.md

```diff
@@ -0,0 +1,50 @@
+title: Sending Docker Logs with Logagent
+description: Collect and ship container logs with Logagent
+
+[Logagent](https://sematext.com/logagent/) is a general purpose open-source log shipper. The [Logagent Docker image](https://hub.docker.com/_/logagent) is pre-configured for[log collection on container platforms](https://sematext.com/blog/docker-container-monitoring-with-sematext/#toc-container-logs-0). It runs as a tiny container on every Docker host and collects logs for all cluster nodes and their containers. Logagent enriches container logs with container metadata such as Kubernetes, Docker Enterprise, and Docker Swarm.
+
+Logagent deployment is very simple. See[ all configurations options](https://sematext.com/docs/logagent/installation-docker/#configuration-parameters). 
+
+Logagent recognizes log formats from various applications / official images out of the box. The [patterns.yml](https://github.com/sematext/logagent-js/blob/master/patterns.yml) file for [log parser patterns](https://sematext.com/docs/logagent/parser/) contains regular expressions to match container name or images, regular expressions to extract fields, field names and optional JavaScript functions to enrich or manipulate the parsed log event. 
+
+The following short example shows how easy it is to deploy Logagent, run a web server, and get structured web server logs for web analytics in Sematext. Logagent ships parsed web server logs via the Elasticsearch API to Sematext Cloud. 
+
+```sh
+
+# Start Logagent
+docker run -d --restart=always -e LOGS_TOKEN=YourLogsToken \
```

Comment:
We have used different names for this Logs Token variable. Let's normalize it and use `<LOGS_TOKEN>` everywhere. This would mean we also need to change all different names of this to `<LOGS_TOKEN>` in all `/docs/logs` sections.

Reply:
I've seen in the rest of the docs, a more usual name for this token is `<YOUR_LOGS_APP_TOKEN_HERE>`. Let's use this instead. Sorry for the constant pinging... :confused: